.vscode/settings.json

```diff
@@ -1,5 +1,11 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
-  "editor.formatOnPaste": false
+  "editor.formatOnPaste": false,
+  "[typescript]": {
+    "editor.defaultFormatter": "vscode.typescript-language-features"
```

Comment:
how does this affect the eslint/prettier linting/formatting?

Reply:
https://github.com/prettier/prettier-vscode/issues/388#issuecomment-489067554

suggests this overrides/disables Prettier

Reply:
quick testing on branch seems to have disabled prettier